app/controllers/outgoings_controller.rb

```diff
@@ -4,21 +4,12 @@ class OutgoingsController < ApplicationController
   formats ['json']
   param :assessment_id, :uuid, required: true
 
-  param :outgoings, Hash, required: true do
-    param :childcare, Array, desc: 'Collection of childcare payments made', required: false do
-      param :payment_date, Date, date_option: :today_or_older, required: true, desc: 'The date the childcare payment was made'
-      param :amount, :currency, required: true, desc: 'The monetary amount paid for the childcare'
-    end
-
-    param :maintenance, Array, desc: 'Collection of maintenance payments made', required: false do
-      param :payment_date, Date, date_option: :today_or_older, required: true, desc: 'The date the maintenance payment was made'
-      param :amount, :currency, required: true, desc: 'The monetary value of the maintenance payment'
-    end
-
-    param :housing_costs, Array, desc: 'Collection of housing cost payment', required: false do
-      param :payment_date, Date, date_option: :today_or_older, required: true, desc: 'The date the housing cost payment was made'
-      param :amount, :currency, required: true, desc: 'The monetary amount paid for the housing cost'
-      param :housing_cost_type, String, required: true, desc: 'Type of housing cost, one of: rent, mortgage, board_and_lodging'
+  param :outgoings, Array, desc: 'Collection of other regular income sources' do
```

Comment:
spelling error, assuming it's '...regular outgoing sources' rather than '...income sources' ?

Also, below it, is it 'The date payment made' rather than 'The date payment received' ?

Reply:
Yep, thanks.